src/components/footer/_macro.njk

```diff
@@ -221,7 +221,7 @@
                 {% if params.copyrightDeclaration is defined and params.copyrightDeclaration %}
                 <div class="grid">
                     <div class="grid__col u-mt-m">
-                        <p class="u-fs-s u-mb-no">&copy; {{ params.copyrightDeclaration.copyright }} <br> {{ params.copyrightDeclaration.text }} <a href="{{ params.copyrightDeclaration.url }}">{{ params.copyrightDeclaration.link }}</a>.</p>
+                        <p class="u-fs-s u-mb-no">&copy; {{ params.copyrightDeclaration.copyright }} <br> {{ params.copyrightDeclaration.text }} <a href="{{ params.copyrightDeclaration.url }}" target="{{ params.copyrightDeclaration.target }}">{{ params.copyrightDeclaration.link }}</a>.</p>
```

Comment:
Does this need an `if` statement in case target is not defined?

Reply:
Will need `rel="noopener"` if `target` is defined, so could bundle both attributes with `params.newWindow` like we do in button component...
```
if params.newWindow is defined and params.newWindow %}target="_blank" rel="noopener"{% endif %}
```

Reply:
yep ill sort that out